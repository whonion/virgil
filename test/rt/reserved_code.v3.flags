-reserved-code-size=4K
