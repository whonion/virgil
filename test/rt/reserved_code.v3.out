##+checking code
##-fail: machine code does not match expectation
